src/__tests__/resources/events.integration.spec.ts

```diff
@@ -95,6 +95,19 @@ describe('Events integration test', () => {
     expect(aggregates[0].count).toBeDefined();
   });
 
+  test('values aggregate', async () => {
+    const aggregates = await client.events.aggregate({
+      filter: {
+        source: 'WORKMATE',
+      },
+      fields: ['type'],
+      aggregate: 'values',
```

Comment:
Not a fan of the usability here. The user has to know what fields are compatible with each aggregate type.

Reply:
but it's typed, you cannot randomly choose a string